### ISSUE TITLE
[bitnami/wordpress] Mounted files are corrupted when using Ubuntu18 with Apache 

### ISSUE THREAD

**Issue by aks-wp** [bitnami/charts#5535]
Issue:

We recently upgraded our Azure Kubernetes clusters to v18 which uses Ubuntu18 and our Wordpress image uploads stopped working. In troubleshooting with MSFT support, looks like the issue boils down to this bug https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=900821 

The following solution in that thread appears to be a reasonable workaround. 
> by default, apache uses mmap, so probably mmap is broken on cifs. An
alternate workaround should be to set EnableMMAP off in the apache
config.

Reference: https://httpd.apache.org/docs/2.4/mod/core.html#enablemmap 

Question:
Since this a **critical** issue for users with clusters on Ubuntu18, is there a clean way to implement the workaround. Any other alternate recommendations are also welcome. 

Additional Information:

Chart: 10.6.5 (latest)
Mount: Azure Files

**Comment by aks-wp** [comment]
Hi @rafariossaa, looks like a custom httpd.conf can be set based on https://github.com/bitnami/bitnami-docker-apache#full-configuration. If there is a way to update one specific field that might be ideal but if a custom file needs to be used, I see the error
EROFS: read-only file system, open '/opt/bitnami/apache/conf/httpd.conf'

**Comment by javsalgar** [comment]
Hi,

Could you show the full logs to know the point where the error appears?

**Comment by aks-wp** [comment]
app-entrypoint.sh 18:47:19.50 
app-entrypoint.sh 18:47:19.50 Welcome to the Bitnami wordpress container
app-entrypoint.sh 18:47:19.51 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-wordpress
app-entrypoint.sh 18:47:19.51 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-wordpress/issues
app-entrypoint.sh 18:47:19.51 
nami    INFO  Initializing apache
Error executing 'postInstallation': EROFS: read-only file system, open '/opt/bitnami/apache/conf/httpd.conf'
nami    TRACE Error: Error executing 'postInstallation': EROFS: read-only file system, open '/opt/bitnami/apache/conf/httpd.conf'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.writeFileSync (fs.js:1299:33)
    at write (/opt/bitnami/nami/node_modules/nami-utils/lib/file/write.js:23:8)
    at /opt/bitnami/nami/node_modules/nami-utils/lib/file/substitute.js:45:7
    at r (/opt/bitnami/nami/node_modules/lodash/index.js:5:348)
    at Function.sf (/opt/bitnami/nami/node_modules/lodash/index.js:78:256)
    at _massFileSubstitute (/opt/bitnami/nami/node_modules/nami-utils/lib/file/substitute.js:39:5)
    at Object.substitute (/opt/bitnami/nami/node_modules/nami-utils/lib/file/substitute.js:180:5)
    at Object.wrappedFn (/opt/bitnami/nami/node_modules/nami-utils/lib/function-wrapping.js:191:17)
    at Object.$app.helpers.setupPorts (/.nami/components/com.bitnami.apache/helpers.js:42:11)

**Comment by aks-wp** [comment]
Hi @javsalgar, For the above test, used the following...

In values.yaml 
  ```
extraVolumeMounts:
     - name: wordpress-config-volume
       mountPath: /opt/bitnami/apache/conf/httpd.conf
       subPath: httpd.conf

  extraVolumes:
    - name: wordpress-config-volume
      configMap:
        name: wordpress-configmap
        defaultMode: 0777
```
ConfigMap
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: wordpress-configmap
data:
  httpd.conf: |
<<content of existing httpd.conf with update to ENABLEMMAP OFF>>
```

**Comment by javsalgar** [comment]
Hi,

I think I found why the read-only filesystem error appears. I will work on adding a fix to this. I will let you know as soon as it available.

**Comment by aks-wp** [comment]
Appreciate you looking into this @javsalgar !

**Comment by marcosbc** [comment]
Hi all, we just released WordPress chart 11.0.0 with [major changes to the container image](https://github.com/bitnami/charts/tree/master/bitnami/wordpress#1100), could you try it and let us know if it continues failing?
